TRACING.md

```diff
@@ -21,7 +20,7 @@ impossible.
 ## How is tracing configured?
 
 You must supply configuration and credentials for the APM server (see below).
-In your `elasticsearch.yml` add the following configuration:
+In your `elasticsearch.yml` add the following configuration. Note that the
```

Comment:
Is this a mistake?

Reply:
Argh, yes, I was going to expand a point but then...didn't.